<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNamedPositionList</class>
 <widget class="QDialog" name="DialogNamedPositionList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>坐标点管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="text">
    <string>修改</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>移动到此为止</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>设为上料位置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNamedPositionList</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNamedPositionList</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
